Documentation/airflow.md

```diff
@@ -0,0 +1,103 @@
+# Airflow Overview
```

Comment:
You need to create an entry for this document on [index.rst](https://github.com/thegraphnetwork/EpiGraphHub/blob/main/Documentation/index.rst) Perhaps named: "Creating Data Collection DAGS". Maybe it can replace [this page](https://github.com/thegraphnetwork/EpiGraphHub/blob/main/Documentation/datasources.md), but first take a look at it and move any still relevant information to this page.